automation/terraform/monitoring/README.md

```diff
@@ -0,0 +1,181 @@
+<p><img src="https://storage.googleapis.com/coda-charts/Mina_Icon_Secondary_RGB_Black.png" alt="Mina logo" title="mina" align="left" height="60" /></p>
+
+# Testnet Monitoring & Alerting :fire_engine: Guide
+
+**Table of Contents**
+- [Updating Testnet Alerts](#update-testnet-alerts)
+    - [Developing alert expressions](#developling)
+    - [Testing](#testing)
+    - [Deployment](#deployment)
+- [Alert Status](#alert-status)
+    - [GrafanaCloud Config](#grafancloud-config)
+    - [Alertmanager UI](#alertmanager-ui)
+    - [PagerDuty](#pagerduty)
+- [HowTo](#howto)
+    - [Silence Alerts](#silence-alerts)
+    - [Update Alert Receivers](#update-alert-receivers)
+    - [View Alert Metrics](#view-alert-metrics)
+
+## Updating Testnet Alerts
+
+#### Developing alert expressions
+
+Developing alert expressions consists of using Prometheus's domain-specific [query language](https://prometheus.io/docs/prometheus/latest/querying/basics/) ([examples](https://prometheus.io/docs/prometheus/latest/querying/examples/)) coupled with its alert rules specification [format](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/) for devising metric and alerting conditions/rules.
```

Comment:
```suggestion
You can developing alert expressions by using Prometheus's domain-specific [query language](https://prometheus.io/docs/prometheus/latest/querying/basics/) ([examples](https://prometheus.io/docs/prometheus/latest/querying/examples/)) coupled with its alert rules specification [format](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/) for devising metric and alerting conditions/rules.
```

Reply:
@dkijania let's use the second person "you" to help our readers 

I follow this guidance https://developers.google.com/style/person

A friend recently wrote https://www.knowledgeowl.com/blog/posts/customer-first-in-your-sentences which helped me remember!